techhive/server.js

```diff
@@ -113,6 +116,34 @@ app.post(
       const internship = req.body;
       internship.picture = req.file.filename; // Assigning the saved file name to the picture field
       const newInternship = await Internship.create(internship);
+
+      // Get all student users' email addresses from the database
+      const studentUsers = await User.findAll({
+        where: {
+          accountType: 'student',
+        },
+      });
```

Comment:
currently we're sending an email to all student accounts every time a new job posting is posted.

I'm assuming this logic will be changed in the future to be personalized to specific students (whether it's their saved internships or preferences)?